{"uid":"4a8f3cacd5cda1da","name":"test_delete_user_authorized_as_another_user","fullName":"tests.test_user_delete.TestUserDelete#test_delete_user_authorized_as_another_user","historyId":"f921dd3d5d139c34912ef23ea50982dc","time":{"start":1709874472893,"stop":1709874477504,"duration":4611},"description":"This test verifies that one user cannot delete another user","descriptionHtml":"<p>This test verifies that one user cannot delete another user</p>\n","status":"passed","statusMessage":"XPASS This test is expected to fail due to a known defect","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[],"testStage":{"description":"This test verifies that one user cannot delete another user","status":"passed","statusMessage":"XPASS This test is expected to fail due to a known defect","steps":[{"name":"POST request to URL '/user/'","time":{"start":1709874472893,"stop":1709874473854,"duration":961},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"POST request to URL '/user/'","time":{"start":1709874473854,"stop":1709874474811,"duration":957},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"POST request to URL '/user/login'","time":{"start":1709874474812,"stop":1709874475322,"duration":510},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"DELETE request to URL '/user/94145'","time":{"start":1709874475322,"stop":1709874476440,"duration":1118},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"GET request to URL '/user/94145'","time":{"start":1709874476440,"stop":1709874477501,"duration":1061},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":5,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},"afterStages":[],"labels":[{"name":"epic","value":"User Deletion Tests"},{"name":"tag","value":"@pytest.mark.xfail(reason='This test is expected to fail due to a known defect')"},{"name":"parentSuite","value":"tests"},{"name":"suite","value":"test_user_delete"},{"name":"subSuite","value":"TestUserDelete"},{"name":"host","value":"fv-az1532-358"},{"name":"thread","value":"2200-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.test_user_delete"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[],"history":{"statistic":{"failed":0,"broken":0,"skipped":151,"passed":1,"unknown":0,"total":152},"items":[{"uid":"580b717d8d7cf28","reportUrl":"https://Badx86.github.io/LearnQA_Python_API/153//#testresult/580b717d8d7cf28","status":"skipped","statusDetails":"XFAIL This test is expected to fail due to a known defect\n\nAssertionError: Unexpected status code 200\nassert 200 == 400\n +  where 200 = <Response [200]>.status_code","time":{"start":1709788001418,"stop":1709788005185,"duration":3767}},{"uid":"286da7f526e96654","reportUrl":"https://Badx86.github.io/LearnQA_Python_API/152//#testresult/286da7f526e96654","status":"skipped","statusDetails":"XFAIL This test is expected to fail due to a known defect\n\nAssertionError: Unexpected status code 200\nassert 200 == 400\n +  where 200 = <Response [200]>.status_code","time":{"start":1709701615062,"stop":1709701619021,"duration":3959}},{"uid":"3b793e5fb9920855","reportUrl":"https://Badx86.github.io/LearnQA_Python_API/151//#testresult/3b793e5fb9920855","status":"skipped","statusDetails":"XFAIL This test is expected to fail due to a known defect\n\nAssertionError: Unexpected status code 200\nassert 200 == 400\n +  where 200 = <Response [200]>.status_code","time":{"start":1709615240592,"stop":1709615244448,"duration":3856}},{"uid":"2080d8a043a94bbf","reportUrl":"https://Badx86.github.io/LearnQA_Python_API/150//#testresult/2080d8a043a94bbf","status":"skipped","statusDetails":"XFAIL This test is expected to fail due to a known defect\n\nAssertionError: Unexpected status code 200\nassert 200 == 400\n +  where 200 = <Response [200]>.status_code","time":{"start":1709530884615,"stop":1709530888095,"duration":3480}},{"uid":"9246ba2951849e5d","reportUrl":"https://Badx86.github.io/LearnQA_Python_API/149//#testresult/9246ba2951849e5d","status":"skipped","statusDetails":"XFAIL This test is expected to fail due to a known defect\n\nAssertionError: Unexpected status code 200\nassert 200 == 400\n +  where 200 = <Response [200]>.status_code","time":{"start":1709442428244,"stop":1709442432166,"duration":3922}},{"uid":"b75b37e511f98dd9","reportUrl":"https://Badx86.github.io/LearnQA_Python_API/148//#testresult/b75b37e511f98dd9","status":"skipped","statusDetails":"XFAIL This test is expected to fail due to a known defect\n\nAssertionError: Unexpected status code 200\nassert 200 == 400\n +  where 200 = <Response [200]>.status_code","time":{"start":1709356053858,"stop":1709356057998,"duration":4140}},{"uid":"f788fc51e3364fec","reportUrl":"https://Badx86.github.io/LearnQA_Python_API/147//#testresult/f788fc51e3364fec","status":"skipped","statusDetails":"XFAIL This test is expected to fail due to a known defect\n\nAssertionError: Unexpected status code 200\nassert 200 == 400\n +  where 200 = <Response [200]>.status_code","time":{"start":1709269671408,"stop":1709269675197,"duration":3789}},{"uid":"1047d28714dbad79","reportUrl":"https://Badx86.github.io/LearnQA_Python_API/146//#testresult/1047d28714dbad79","status":"skipped","statusDetails":"XFAIL This test is expected to fail due to a known defect\n\nAssertionError: Unexpected status code 200\nassert 200 == 400\n +  where 200 = <Response [200]>.status_code","time":{"start":1709183309495,"stop":1709183313730,"duration":4235}},{"uid":"465e173f2130a36e","reportUrl":"https://Badx86.github.io/LearnQA_Python_API/145//#testresult/465e173f2130a36e","status":"skipped","statusDetails":"XFAIL This test is expected to fail due to a known defect\n\nAssertionError: Unexpected status code 200\nassert 200 == 400\n +  where 200 = <Response [200]>.status_code","time":{"start":1709096894001,"stop":1709096898037,"duration":4036}},{"uid":"95438340a0f0212b","reportUrl":"https://Badx86.github.io/LearnQA_Python_API/144//#testresult/95438340a0f0212b","status":"skipped","statusDetails":"XFAIL This test is expected to fail due to a known defect\n\nAssertionError: Unexpected status code 200\nassert 200 == 400\n +  where 200 = <Response [200]>.status_code","time":{"start":1709010486181,"stop":1709010489433,"duration":3252}},{"uid":"5e3d54345afe6a21","reportUrl":"https://Badx86.github.io/LearnQA_Python_API/143//#testresult/5e3d54345afe6a21","status":"skipped","statusDetails":"XFAIL This test is expected to fail due to a known defect\n\nAssertionError: Unexpected status code 200\nassert 200 == 400\n +  where 200 = <Response [200]>.status_code","time":{"start":1708924068982,"stop":1708924073403,"duration":4421}},{"uid":"6e31e1d9fca6b49","reportUrl":"https://Badx86.github.io/LearnQA_Python_API/142//#testresult/6e31e1d9fca6b49","status":"skipped","statusDetails":"XFAIL This test is expected to fail due to a known defect\n\nAssertionError: Unexpected status code 200\nassert 200 == 400\n +  where 200 = <Response [200]>.status_code","time":{"start":1708837547590,"stop":1708837551282,"duration":3692}},{"uid":"2d1a566a48c6a5cf","reportUrl":"https://Badx86.github.io/LearnQA_Python_API/141//#testresult/2d1a566a48c6a5cf","status":"skipped","statusDetails":"XFAIL This test is expected to fail due to a known defect\n\nAssertionError: Unexpected status code 200\nassert 200 == 400\n +  where 200 = <Response [200]>.status_code","time":{"start":1708751234450,"stop":1708751239962,"duration":5512}},{"uid":"df5706e985c111e3","reportUrl":"https://Badx86.github.io/LearnQA_Python_API/140//#testresult/df5706e985c111e3","status":"skipped","statusDetails":"XFAIL This test is expected to fail due to a known defect\n\nAssertionError: Unexpected status code 200\nassert 200 == 400\n +  where 200 = <Response [200]>.status_code","time":{"start":1708664871326,"stop":1708664875153,"duration":3827}},{"uid":"2a7f1b3e373b6ed3","reportUrl":"https://Badx86.github.io/LearnQA_Python_API/139//#testresult/2a7f1b3e373b6ed3","status":"skipped","statusDetails":"XFAIL This test is expected to fail due to a known defect\n\nAssertionError: Unexpected status code 200\nassert 200 == 400\n +  where 200 = <Response [200]>.status_code","time":{"start":1708578432574,"stop":1708578437331,"duration":4757}},{"uid":"e163d14e0de94c38","reportUrl":"https://Badx86.github.io/LearnQA_Python_API/138//#testresult/e163d14e0de94c38","status":"skipped","statusDetails":"XFAIL This test is expected to fail due to a known defect\n\nAssertionError: Unexpected status code 200\nassert 200 == 400\n +  where 200 = <Response [200]>.status_code","time":{"start":1708492006385,"stop":1708492009889,"duration":3504}},{"uid":"9ed061dc5d9842ec","reportUrl":"https://Badx86.github.io/LearnQA_Python_API/137//#testresult/9ed061dc5d9842ec","status":"skipped","statusDetails":"XFAIL This test is expected to fail due to a known defect\n\nAssertionError: Unexpected status code 200\nassert 200 == 400\n +  where 200 = <Response [200]>.status_code","time":{"start":1708405671577,"stop":1708405675021,"duration":3444}},{"uid":"487bc86cfa0a4485","reportUrl":"https://Badx86.github.io/LearnQA_Python_API/136//#testresult/487bc86cfa0a4485","status":"skipped","statusDetails":"XFAIL This test is expected to fail due to a known defect\n\nAssertionError: Unexpected status code 200\nassert 200 == 400\n +  where 200 = <Response [200]>.status_code","time":{"start":1708319275069,"stop":1708319278949,"duration":3880}},{"uid":"20857496911fd5d7","reportUrl":"https://Badx86.github.io/LearnQA_Python_API/135//#testresult/20857496911fd5d7","status":"skipped","statusDetails":"XFAIL This test is expected to fail due to a known defect\n\nAssertionError: Unexpected status code 200\nassert 200 == 400\n +  where 200 = <Response [200]>.status_code","time":{"start":1708232752924,"stop":1708232756446,"duration":3522}},{"uid":"c66703bfb8266b","reportUrl":"https://Badx86.github.io/LearnQA_Python_API/134//#testresult/c66703bfb8266b","status":"skipped","statusDetails":"XFAIL This test is expected to fail due to a known defect\n\nAssertionError: Unexpected status code 200\nassert 200 == 400\n +  where 200 = <Response [200]>.status_code","time":{"start":1708146398226,"stop":1708146402211,"duration":3985}}]},"tags":["@pytest.mark.xfail(reason='This test is expected to fail due to a known defect')"]},"source":"4a8f3cacd5cda1da.json","parameterValues":[]}